### PR TITLE
Feature/HOLO-642: Implement Super Cold Storage logic into protocol

### PR DESCRIPTION
**Pull request by ACC01ADE**
## Describe Changes

Adding support for the Super Cold Storage device (SCS).

- Provided a way to activate via `.env` file
- Modified deployment pipeline to support the SCS device.
- Using NPM package [super-cold-storage-signer](https://github.com/ACC01ADE/super-cold-storage-signer)
- Updated package [@holographxyz/hardhat-deploy-holographed](https://github.com/holographxyz/hardhat-deploy-holographed)
- Using code from https://github.com/ACC01ADE/super-cold-storage-nodejs for device server
- Using code from https://github.com/ACC01ADE/yubihsm-shell-keccak256 for enabling keccak256 hashes of YubiHSM module
- Using code from https://github.com/ACC01ADE/secp256k1-pem for creating keys to test against protocol

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

File: contracts/enforcer/HolographRoyalties.sol [hunk 1]
```diff
@@ -192,7 +192,7 @@ contract HolographRoyalties is Admin, Owner, Initializable {
     uint256 bp = abi.decode(initPayload, (uint256));
     setRoyalties(0, payable(address(this)), bp);
     address payable[] memory addresses = new address payable[](1);
-    addresses[0] = payable(getOwner());
+    addresses[0] = payable(Owner(HolographerInterface(address(this)).getSourceContract()).owner());
```
Comment (sogoiii) [inline contracts/enforcer/HolographRoyalties.sol:195]: @ACC01ADE Is this change required given we merged in @alexanderattar PR?

**Review by sogoiii — APPROVED** [review]
👍

**Review by alexanderattar — APPROVED** [review]
👍
